code-compass.el

```diff
@@ -328,280 +331,287 @@
    lines))
 
-(defun c/find-indentation (lines-without-text)
+(defun code-compass--find-indentation (lines-without-text)
   "Infer indentation level in LINES-WITHOUT-TEXT. If no indentation present in file, defaults to 2."
   (or (--> lines-without-text
         (--map (list (s-count-matches "\s" it) (s-count-matches "\t" it)) it)
-        (let ((spaces-ind (-sort '< (--remove (eq 0 it) (-map 'c/first it))))
-              (tabs-ind (-sort '< (--remove (eq 0 it) (-map 'c/second it)))))
+        (let ((spaces-ind (-sort '< (--remove (eq 0 it) (-map 'code-compass--first it))))
+              (tabs-ind (-sort '< (--remove (eq 0 it) (-map 'code-compass--second it)))))
           (if (> (length spaces-ind) (length tabs-ind))
-              (c/first spaces-ind)
-            (c/first tabs-ind))))
+              (code-compass--first spaces-ind)
+            (code-compass--first tabs-ind))))
       2))
 
-(defun c/convert-tabs-to-spaces (line-without-text n)
+(defun code-compass--convert-tabs-to-spaces (line-without-text n)
   "Replace tabs in LINE-WITHOUT-TEXT with N spaces."
   (s-replace "\t" (make-string n ?\s) line-without-text))
 
-(defun c/calculate-complexity (line-without-text indentation)
+(defun code-compass--calculate-complexity (line-without-text indentation)
   "Calculate indentation complexity by dividing length of LINE-WITHOUT-TEXT by INDENTATION."
   (/ (+ 0.0 (length line-without-text)) indentation))
 
-(defun c/as-logical-indents (lines &optional opts)
+(defun code-compass--as-logical-indents (lines &optional opts)
   "Calculate logical indentations of LINES. Try to infer how many space is an indent unless OPTS provides it."
-  (let ((indentation (or opts (c/find-indentation lines))))
+  (let ((indentation (or opts (code-compass--find-indentation lines))))
     (list
      (--map
       (--> it
-        (c/convert-tabs-to-spaces it indentation)
-        (c/calculate-complexity it indentation))
+        (code-compass--convert-tabs-to-spaces it indentation)
+        (code-compass--calculate-complexity it indentation))
       lines)
      indentation)))
 
-(defun c/stats-from (complexities-indentation)
+(defun code-compass--stats-from (complexities-indentation)
   "Return stats from COMPLEXITIES-INDENTATION."
-  (let* ((complexities (c/first complexities-indentation))
+  (let* ((complexities (code-compass--first complexities-indentation))
          (mean (/ (-sum complexities) (length complexities)))
          (sd (sqrt (/ (-sum (--map (expt (- it mean) 2) complexities)) (length complexities)))))
     `((total . ,(-sum complexities))
       (n-lines . ,(length complexities))
       (max . ,(-max complexities))
       (mean . ,mean)
       (standard-deviation . ,sd)
-      (used-indentation . ,(c/second complexities-indentation)))))
+      (used-indentation . ,(code-compass--second complexities-indentation)))))
 
-(defun c/calculate-complexity-stats (code &optional opts)
```

Comment:
also this one: could we alias and leave public API? `code-compass-calculate-complexity-stats`

This is the last of those I am certain are used somewhere else.

Reply:
done